projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -97,6 +109,168 @@ public static function is_banner_enabled() {
 		return $recommendations_banner_enabled;
 	}
 
+	/**
+	 * Set up actions to monitor for things that trigger a recommendation.
+	 *
+	 * @return false|void
+	 */
+	public static function init_conditional_recommendation_actions() {
+		// Check to make sure that recommendations are enabled.
+		if ( ! self::is_enabled() ) {
+			return false;
+		}
+
+		// Monitor for the publishing of a new post.
+		add_action( 'transition_post_status', array( get_called_class(), 'post_transition' ), 10, 3 );
+		add_action( 'jetpack_activate_module', array( get_called_class(), 'jetpack_module_activated' ), 10, 2 );
```

Comment:
`get_called_class()` is a new one for me. Just curious, would `self::class` work here the same?